examples/01_Simulation.ipynb

```diff
@@ -392,6 +392,22 @@
     "print('Last timestep: ', results.times[-1])"
    ]
   },
+  {
```

Comment:
Suggestion: What if right before this we add a line of text explaining the 'all' simulation? Here's what I'm thinking - a few lines up, we simulate without specifying event_strategy, and we note that the simulation ends at 3.8 sec. Now we are simulating using event_strategy='all' and printing the last time which is 7.89 sec. We could add a line of text here highlighting the increase in time step now that the simulation runs until all events have occurred, just to make things clear.

Reply:
Ah, now I see that this text was there, but is now below the event_strategy = thresholds_met_eqn code block. What if we include the above, and then also include a line of text after the thresholds_met_eqn block that describes that result too

Reply:
Updated